openhtf/util/__init__.py

```diff
@@ -117,3 +117,22 @@ def get_version():
     version = 'Unknown - Perhaps openhtf was not installed via setup.py or pip.'
 
   return version
+
+
+class NonLocalResult(mutablerecords.Record('NonLocal', {'result': None})):
```

Comment:
Even if we don't cache dut_id in triggers.py, this is handy for phase_result, if you want to keep it and use it there instead ;)